eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           NuGetCacheKey: Test
       - script: >-
           dotnet test eng/service.proj --filter "(TestCategory!=Manually) & (TestCategory!=Live)" --framework $(TestTargetFramework)
-          --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal"
+          --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal" --blame-crash
```

Comment:
If this is helpful we might want to add it to live test command as well. Also nit but I'd suggest putting it on its own line.